src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const queryLowerCase = query.trim().toLowerCase();
+  const visibleMovies: Movie[] = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(queryLowerCase)
+      || movie.description.toLowerCase().includes(queryLowerCase),
```

Comment:
de-structuring + concat title and description could make it bit shorter
```suggestion
    { title, description } => (title + description).toLowerCase().includes(queryLowerCase)
```